src/Groups/directproducts.jl

```diff
@@ -415,6 +430,17 @@ function (W::WreathProductGroup)(L::Union{GAPGroupElem{T},GAPGroupElem{PermGroup
    return group_element(W,xgap)
 end
 
+"""
+    factor_of_wreath_product(G::WreathProductGroup, j::Int)
+If `W` is the wreath product of `G` and `H`, return `G` (if `j`=1) or `H` (if `j`=2).
```

Comment:
That's completely unclear! What is G and what is H ???? Which one is the one which acts, which one is the one which ends up as component of the normal subgroup? At the very least this should inform the reader that they need to look at the documentation for `wreath_product` to understand this.